api/envoy/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto

```diff
@@ -551,7 +551,8 @@ message HttpConnectionManager {
   // company-internal mesh) and false when receiving untrusted traffic (edge deployments).
   //
   // If different behaviors for invalid_http_message for HTTP/1 and HTTP/2 are
-  // desired, one *must* use the new HTTP/2 option
+  // desired, one *must* use the new HTTP/1 option :ref:`override_stream_error_on_invalid_http_message
```

Comment:
I think we can lose the emphasis on *must* now :-)

 and the new HTTP/2 option -> or the new HTTP/2 option